Week-4/Promises.js

```diff
@@ -0,0 +1,25 @@
+function getNumber() {
+  return Math.floor(Math.random() * 10);
+}
+let storedNumber = getNumber();
+let dividedNumber = storedNumber % 5;
+// console.log(getNumber())
+console.log(storedNumber);
+// console.log(dividedNumber)
+let checkNumber = new Promise(function (resolve, reject) {
+  setTimeout(() => {
+    if (dividedNumber != 0) {

```

Comment:
No point of using curly braces for single statement. Use
if (dividedNumber != 0)  resolve(storedNumber);